### PR TITLE
msm8974: Add flag to prevent an SDIO device in the second slot to be …

### PR DESCRIPTION
**Pull request by thefat32**
…initialized wrong by the MMC driver (this happens in hammerhead MSM8974) and SDIO Wifi Chip stucks in a bad HW State. (This is a simple workaround for the problem - someone with better SDIO understanding could probably code a runtime solution for this)

Signed-off-by: Lucas Picchi <picchi.lucas@gmail.com>

### CODE REVIEW

File: target/msm8974/init.c [hunk 1]
```diff
@@ -274,7 +275,7 @@ static void target_mmc_sdhci_init(void)
 	}
 	else if(!dev)
 		dev = tmpdev;
-
```
Comment (M1cha) [inline target/msm8974/init.c]: why did you remove this empty line? :)
just move it behind the endif

**Review by M1cha — CHANGES_REQUESTED** [review]

**Comment by thefat32** [comment]
Done
